include/SDL3/SDL_time.h

```diff
@@ -0,0 +1,205 @@
+   claim that you wrote the original software. If you use this software
+   in a product, an acknowledgment in the product documentation would be
+   appreciated but is not required.
+2. Altered source versions must be plainly marked as such, and must not be
+   misrepresented as being the original software.
+3. This notice may not be removed or altered from any source distribution.
+*/
+
+#ifndef SDL_time_h_
+#define SDL_time_h_
+
+/**
+ *  \file SDL_time.h
+ *
+ *  Header for the SDL realtime clock and date/time routines.
+ */
+
+#include <SDL3/SDL_error.h>
+#include <SDL3/SDL_stdinc.h>
+
+#include <SDL3/SDL_begin_code.h>
+/* Set up for C function definitions, even when using C++ */
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ *
+ * A structure holding a calendar date and time broken down into its components.
+ */
+typedef struct SDL_DateTime
+{
+    int year;                  /**< Year */
+    int month;                 /**< Month [01-12] */
+    int day;                   /**< Day of the month [01-31] */
+    int hour;                  /**< Hour [0-23] */
+    int minute;                /**< Minute [0-59] */
+    int second;                /**< Seconds [0-60] */
+    int nanosecond;            /**< Nanoseconds [0-999999999] */
+    int day_of_week;           /**< Day of the week [0-6] (0 being Sunday) */
+    int utc_offset;            /**< Seconds east of UTC */
+} SDL_DateTime;
+
+/**
+ *
+ * The preferred date format of the current system locale.
+ *
+ * \sa SDL_GetDateTimeLocalePreferences
+ */
+typedef enum SDL_DATE_FORMAT
+{
+    SDL_DATE_FORMAT_YYYYMMDD = 0, /**< Year/Month/Day */
+    SDL_DATE_FORMAT_DDMMYYYY = 1, /**< Day/Month/Year */
+    SDL_DATE_FORMAT_MMDDYYYY = 2, /**< Month/Day/Year */
+} SDL_DATE_FORMAT;
+
+/**
+ *
+ * The preferred time format of the current system locale.
+ *
+ * \sa SDL_GetDateTimeLocalePreferences
+ */
+typedef enum SDL_TIME_FORMAT
+{
+    SDL_TIME_FORMAT_24HR = 0, /**< 24 hour time */
+    SDL_TIME_FORMAT_12HR = 1, /**< 12 hour time */
+} SDL_TIME_FORMAT;
+
+/**
+ *  Global date/time properties
+ *
+ *  - `SDL_PROP_GLOBAL_SYSTEM_DATE_FORMAT_NUMBER`: the SDL_DATE_FORMAT to use as the preferred date display format
+ *    for the current system locale.
+ *  - `SDL_PROP_GLOBAL_SYSTEM_TIME_FORMAT_NUMBER`: the SDL_TIME_FORMAT to use as the preferred time display format
+ *    for the current system locale.
+ */
+#define SDL_PROP_GLOBAL_SYSTEM_DATE_FORMAT_NUMBER "SDL.time.date_format"
+#define SDL_PROP_GLOBAL_SYSTEM_TIME_FORMAT_NUMBER "SDL.time.time_format"
+
+/**
+ *
+ * Gets the current value of the system realtime clock in nanoseconds since Jan 1, 1970 UTC.
+ *
+ * \param ticks the SDL_Time to hold the returned tick count
+ * \returns 0 on success or -1 on error; call SDL_GetError() for more information.
+ *
+ * \since This function is available since SDL 3.0.0 
+ */
+extern DECLSPEC int SDLCALL SDL_GetRealtimeClockTicks(SDL_Time *ticks);
```

Comment:
How do you feel about calling this `SDL_GetCurrentTime()`?

Reply:
That works.